### PR TITLE
fix(examples): storybook publishes

### PR DESCRIPTION
**Pull request by wittjosiah**
copilot:all

### CODE REVIEW

File: packages/sdk/examples/src/examples/TaskList.tsx [hunk 1]
```diff
@@ -16,9 +16,12 @@ import { Task } from '../proto/gen/schema';
 const TaskList = ({ spaceKey, id }: { spaceKey: PublicKey; id: number }) => {
   const space = useSpace(spaceKey);
   const tasks = useQuery(space, Task.filter());
-  const [input, setInput] = useState<HTMLInputElement>();
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  console.log({ id, spaceKey, space, tasks });
```
Comment (thure) [inline packages/sdk/examples/src/examples/TaskList.tsx:21]: 👀

File: packages/sdk/react-client/src/testing/ClientRepeater.tsx [hunk 2]
```diff
@@ -21,41 +25,102 @@ export type ClientRepeaterProps<P extends RepeatedComponentProps> = {
...
  * Utility component for Storybook stories which sets up clients for n peers.
  * The `Component` property is rendered n times, once for each peer.
  */
-// TODO(burdon): Reconcile with ClientSpaceDecorator.
+// TODO(wittjosiah): Rename.
 export const ClientRepeater = <P extends RepeatedComponentProps>(props: ClientRepeaterProps<P>) => {
-  const { clients, count = 1, className = 'flex w-full place-content-evenly', Component } = props;
+  const {
+    count = 1,
+    className = 'flex w-full place-content-evenly',
```
Comment (thure) [inline packages/sdk/react-client/src/testing/ClientRepeater.tsx:44]: If this is getting passed to `mx` the prop name should be `classNames` and the type should be `ClassNameValue`, also available as `ThemedClassName<E extends HTMLElement>`.

**Review by thure — APPROVED** [review]
Thank you for fixing all this!

File: packages/sdk/react-client/src/testing/ClientRepeater.tsx (hunk 2 above)
Comment (wittjosiah) [reply packages/sdk/react-client/src/testing/ClientRepeater.tsx:44]: I think its only being passed to the div on line 90

**Pull request merged**